### PR TITLE
Change stake subsidy minting and update constants values

### PR DESCRIPTION
**Pull request by emmazzz**

### CODE REVIEW

File: crates/sui-framework/sources/governance/stake_subsidy.move [hunk 1]
```diff
@@ -31,6 +31,16 @@ module sui::stake_subsidy {
         }
     }
 
+    public(friend) fun mint_stake_subsidy_proportional_to_total_stake(
+        subsidy: &mut StakeSubsidy, supply: &mut Supply<SUI>, stake_subsidy_rate: u64, total_stake: u64
+    ) {
+        let amount_to_mint = (total_stake as u128) * (stake_subsidy_rate as u128) / BASIS_POINT_DENOMINATOR;
```
Comment (lxfind) [inline crates/sui-framework/sources/governance/stake_subsidy.move:37]: Where does the subsidy come from?

File: crates/sui-protocol-constants/src/lib.rs [hunk 2]
```diff
@@ -105,17 +105,19 @@ pub const OBJ_METADATA_COST_NON_REFUNDABLE: u64 = 50;
...
 pub const REWARD_SLASHING_THRESHOLD_BPS: u64 = 5000;
 
-// TODO: placeholder value here
-// The share of rewards that will be slashed and redistributed is 10%.
-pub const REWARD_SLASHING_RATE: u64 = 1000;
+/// The share of rewards that will be slashed and redistributed is 50%.
+pub const REWARD_SLASHING_RATE: u64 = 5000;
+
+/// How much stake subsidy do we mint each epoch proportional to the total stake.
+pub const STAKE_SUBSIDY_RATE: u64 = 10;
```
[the remark this answers is not in the document]
Comment (emmazzz) [reply crates/sui-protocol-constants/src/lib.rs:123]: basis point. will document

File: crates/sui-framework/sources/governance/stake_subsidy.move (hunk 1 above)
Comment (lxfind) [reply crates/sui-framework/sources/governance/stake_subsidy.move:37]: Please rename this function by appending _testnet

Comment (sblackshear) [reply crates/sui-framework/sources/governance/stake_subsidy.move:37]: Although it has the same semantics, I think it's good to do
```suggestion
        let amount_to_mint = ((total_stake as u128) * (stake_subsidy_rate as u128)) / BASIS_POINT_DENOMINATOR;
```

to make it clear that multiplication before division is important to minimize rounding error.

**Review by sblackshear — APPROVED** [review]
Accepting to unblock pending resolution of Xun's comments.

**Pull request merged**